python/demo/poisson/demo_poisson.py

```diff
@@ -184,7 +181,9 @@
 
 # Compute solution
 u = Function(V)
-solve(a == L, u, bc, petsc_options={"ksp_type": "preonly", "pc_type": "lu"})
+solve(a == L, u, bc, petsc_options={"ksp_type": "preonly",
+                                    "pc_type": "lu", "pc_factor_mat_solver_type": "superlu_dist"})
```

Comment:
Why `superlu_dist` here?

Reply:
Reverted - I had some trouble with MUMPS and macOS.